pyked/tests/test_chemked.py

```diff
@@ -17,6 +17,30 @@
 
 warnings.simplefilter('always')
 
+# Create temporary directory for tests that need to create files
+# Taken from http://stackoverflow.com/a/22726782/1569494
+try:
```

Comment:
In what situation will `tempfile` not be available? We only support Python 3.5 and up